bootstrap/v2/pkg/kfapp/kustomize/kustomize.go

```diff
@@ -468,14 +468,19 @@ func (kustomize *kustomize) Init(resources kftypes.ResourceEnum) error {
 	if len(parts) == 2 {
```

Comment:
Isn't this also part of the legacy code path, since the version is now given from the repo value?